recipe/0103-poetry-reading-annotations/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Annotating a Poetry Reading
+id: 103
+layout: recipe
+tags: [audio, presentation, annotation]
+summary: "Use annotations to indicate aspects of the performance of a particular poem."
+---
+
+
+## Use Case
+
+While an audio file of a poetry performance may be divided into a track for each poem, scholars may wish to use annotations to indicate aspects of the performance of a particular poem.  
+
+A researcher might want to annotate the following types of information:
+* structural information (introduction, title, stanzas)
+* points where the performer takes a breath
+* repeated phrases
+
+Since annotations could be available at the same time the manifest is generated, or might be a separate process that references the item manifest, both scenarios are shown.
+
+There is a third use case where manifests are unaware of annotations on them, but the systems that display the item are aware of the annotations and pull them in, using the target block in the annotation.
+
+## Implementation notes
+
+1. This implementation builds off of the [audio example][0002], but adds Web Annotations.
+
+2. This recipe shows two variations of referencing annotations from within a manifest.  The first has the annotations embedded within the manifest file.  The second has a reference to annotations in a separate file.
+
+3. Where we use "RangeSelector" for the annotation target, we could instead use a Media Fragment like "#t=1.23,2.23" appended to the canvas URL. Instead of "source" you'd condense it into the "target" field.  For example:
+
+```
+                "target": {
+                  "source": "http://localhost:4000/recipe/0103-poetry-reading-annotations/manifest1.json/canvas/segment1/canvas/segment1",
+                  "selector": {
+                    "type": "RangeSelector",
+                    "t": "46.734653,47.875068"
+                  }
+```
+
+Could become
+
+```
+                "target": "http://localhost:4000/recipe/0103-poetry-reading-annotations/manifest1.json/canvas/segment1/canvas/segment1#t=46.734653,47.875068"
+```
+
+Both are correct.
+
+4. While the IIIF Specification requires arrays for most values, the W3C Web Annotation Specification does not require arrays, so the following line: 
+
+`"motivation": "commenting",`
+
+is correct, but unusual for a IIIF resource.  It could also be expressed (and must be if there is more than one motivation) as follows:
+
+`"motivation": ["commenting"],`
```

Comment:
If the IIIF spec requires arrays for motivations, should we remove this implementation note altogether?